keras_nlp/models/bloom/bloom_mlp.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+
+
+class BloomMLP(keras.layers.Layer):
```

Comment:
Is there a reason to split this out? I might just keep in in the decoder block, would keep things a little more concise overall.

Reply:
The reason I have split it like that is because it's illustrated in figure.5 in the [paper](https://arxiv.org/pdf/2211.05100.pdf) as a separate block and also in huggingface it's implemented as a separate layer. But I have moved it inside the decoder block anyway.